Description: Fix tta plugin to work with segment with startms
Author: Yang Zhe <yangzhe1990@gmail.com>
Bug: http://bugs.xmms2.xmms.se/view.php?id=2474
Bug-Debian: http://bugs.debian.org/588519

--- a/src/plugins/tta/tta.c
+++ b/src/plugins/tta/tta.c
@@ -267,7 +267,6 @@
 	}
 
 	data->next_boundary -= ret;
-	data->next_boundary = 0;
 
 	return size;
 }
@@ -307,6 +306,7 @@
 
 	/* update the index of the next frame after successful seek */
 	data->nextframe = idx;
+	data->next_boundary = 0;
 
 	return (data->nextframe * data->framelen);
 }
